fivox/eventSource.h

```diff
@@ -90,6 +90,9 @@ class EventSource
      */
     EventValues findEvents( const AABBf& area ) const;
 
+    /** Set bounding box of upcoming events. */
+    void setBoundingBox( const AABBf& boundingBox );
```

Comment:
This needs to be documented better. I guess you use this to override any bounding box calculation made by the event source, or can this bounding box be augmented as events are processed? If this is just a hint the function should be called differently, e.g. setInitialBoundingBox or setBoundingBoxHint.